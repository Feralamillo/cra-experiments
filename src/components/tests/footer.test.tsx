import React from 'react';
import { shallow } from 'enzyme';

import { Footer } from '../footer';

test('renders the component', () => {
  const wrapper = shallow(<Footer />);

  expect(wrapper).toMatchSnapshot();
});
