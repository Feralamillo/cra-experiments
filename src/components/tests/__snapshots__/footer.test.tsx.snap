// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<footer
  className="text-muted py-5"
>
  <div
    className="container"
  >
    <p
      className="float-right"
    >
      <a
        href="!#"
      >
        Back to top
      </a>
    </p>
    <p>
      Album example is Â© Bootstrap, but please download and customize it for yourself!
    </p>
  </div>
</footer>
`;
